content/06.know-your-problem.md

```diff
@@ -1 +1,18 @@
+It's even better to also have access to a subject matter expert who has either collected or analyzed this type of data before! 
+For example, if the main reason why the data were collected was to test the impact of an intervention, it may be the case that a randomized controlled trial was performed.
+However, ethical or other study considerations may make this impossible or impractical, in which case the design may have been an observational one, either prospective or retrospective.
+These designs may also incorporate some amount of matching - for example, cases and controls may be selected so that the age range or weight distribution is similar.
+All of these different designs have different assumptions and caveats, which cannot be ignored during a data analysis.
+Many datasets are now passively collected or do not have a specific design, but even in this case it is important to know how individuals or samples were treated (for example, if all samples are from the same study site, if certain ethnic groups or zip codes are oversampled, if there are differences in processing dates or techniques.)
+Systematic biases can lead to artifacts or "batch effects," which mean that instead of finding correlates with an outcome or grouping of interest, the investigator may find correlates with variables that are not of interest and obtain misleading results [@doi:10.1038/nrg2825].
+Other study design considerations that should not be overlooked include knowing whether a study involves biological or technical replicates or both.
+For example, are some samples collected from the same individuals at different time points? Are those time points before and after some treatment?
+If one assumes that all the samples are independent but that is in fact not the case, a variety of issues may arise, including having a lower effective sample size than expected.
+In general, deep learning has an increased tendency for overfitting, compared to classical methods, due to the large number of parameters being estimated, making issues of adequate sample size even more important (see [Rule 7](https://github.com/Benjamin-Lee/deep-rules/blob/master/content/09.overfitting.md)).
+For a large dataset, overfitting may not be a concern, but the modeling power of deep learning may lead to more spurious correlations and thus incorrect interpretation of results (see [Rule 9](https://github.com/Benjamin-Lee/deep-rules/blob/master/content/11.interpretation.md)).
+Finally, it is important to note that with the exception of very specific cases of unsupervised data analysis, it is generally the case that a molecular or imaging dataset does not have much value without appropriate clinical or demographic data; this must always be balanced with the need to protect patient privacy (see [Rule 10](https://github.com/Benjamin-Lee/deep-rules/blob/master/content/12.privacy.md)). 
```

Comment:
I disagree with this point as currently stated.  Are you assuming human studies in this sentence?  If that's the case, I suggest being more explicit about that and qualifying the statement.

Reply:
> If one assumes that all the samples are independent but that is in fact not the case, a variety of issues may arise, including having a lower effective sample size than expected.

In most machine learning and deep learning applications, we only have one sample (i.e., the dataset that we split into training and test set). Maybe we could rephrase this such that the term "samples" is not used with different meanings. E.g. we could say "data points/examples in the sample" or sth along these lines to be more clear.

I think we should also add an intuitive example (ideally with a reference). Something I can spontaneously think of is a scenario where your sample consists of several drugs candidates have been pre-selected by some filtering method and then used to train a ML-based model. Since all methods are dependent according to the filtering method being used, this system will likely not generalize to the whole population of molecules that might be tested as drug candidates (but have not undergone the filtering). If you apply your ML-based method to new drug candidates that did undergo the same filtering, I don't see an issue with non-independence within the sample.

Reply:
@agitter I do generally work with human studies, so was thinking mostly about those. Although for animals, you may still want things like age, sex, treatment group etc.
@rasbt Yes, the word "sample" can have many meanings and we may want to standardize it for this paper. I meant that if you have multiple biological samples (like blood samples) from the same person they're not really independent (for example, if you draw blood from someone every 3 months to measure gene expression). Also, if you have related people or some other kind of clustering.

Reply:
No worries :). I think in this case  (" if you have multiple biological samples (like blood samples) from the same person ") we could call this "repeated" or "dependent" "measurements" ?

Reply:
> @agitter I do generally work with human studies, so was thinking mostly about those. Although for animals, you may still want things like age, sex, treatment group etc.

Agreed for animals.  I was thinking of yeast, E. coli, and other frequently studied organisms that don't really have "demographics".

Reply:
> Agreed for animals. I was thinking of yeast, E. coli, and other frequently studied organisms that don't really have "demographics".

Is there a good general name? "Meta-data" doesn't really cut it. "Explanatory variables" or "additional covariates?" Then we could give the specific example of clinical and demographic data for humans. My point was that if you just have the molecular or imaging dataset and that's it, it's often not that helpful.